### PR TITLE
Implemented Google Maps API and added Fact Generator 

### PR DESCRIPTION
**Pull request by juliareichel**
-Last week I had to reclone my repo because it "broke" whatever that means.. so for some reason when I did that some files weren't saved. Re-added the fact generator js file

### CODE REVIEW

File: portfolio/src/main/webapp/facts.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const QUOTES =
```
Comment (damargulis) [inline portfolio/src/main/webapp/facts.js:1]: /QUOTES/FACTS?

**Comment by damargulis** [comment]
Looks like you added another commit on top of this -- try to keep features issolated on their own branch/pr.

This pr was called "Changing color and border of fact box", so thats all it should do. THe changes for the map should go into a different pr

File: portfolio/src/main/webapp/facts.js (hunk 1 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/facts.js]: when I copied from my old version it was quotes but now facts, good spot. Fixed

File: portfolio/src/main/webapp/destinations.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/** Creates a map and adds it to the page. */
+function createMap() {
+  var curLocation = {lat:40.5500 , lng: 14.2333};
```
Comment (damargulis) [inline portfolio/src/main/webapp/destinations.js:3]: You can make these lat/long values top-level consts

File: portfolio/src/main/webapp/destinations.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  var map = new google.maps.Map(document.getElementById('map'), {
```
Comment (damargulis) [inline portfolio/src/main/webapp/destinations.js:4]: use let or const instead of var across this file

Comment (juliareichel) [reply portfolio/src/main/webapp/destinations.js:4]: changed

**Pull request merged**